packages/ui/cypress/fixtures/accounts.ts

```diff
@@ -1,3 +1,4 @@
 export const addresses = {
-  Alice: '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY'
+  Alice: '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY',
+  AliceSS58: '0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d'
```

Comment:
I'd suggest something like this.
```suggestion
  AlicePubKey: '0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d',
  AliceKusama:'HNZata7iMYWmk5RvZRTiAsSDhV8366zq2YGb3tLH5Upf74F'
```

Reply:
I'll remove it, not needed if setting everything up in the UI

Reply:
Removed :)